CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ the detailed section referring to by linking pull requests or issues.
 
 #### Minor
 
+- Implemented markdown for asset descriptions
```

Comment:
![grafik](https://github.com/sovity/edc-ui/assets/30114507/99cc2fb0-b484-4489-adf7-81fb1ea4cc72)
![grafik](https://github.com/sovity/edc-ui/assets/30114507/8cdb0d8d-e596-495a-9ffd-b562cf163c95)


I can understand how you limited the size to be exactly the full height. I believe we should make it smaller in height though, maybe a third of the available height, so we do not disregard the asset details and properties below, which should also be the main focus of the dialog. Even with Windows, the Windows 11 scroll bars are so tiny you can barely see then that this area is even scrollable. What do you think?

Reply:
what do you think?

Reply:
looks good